rootfs/api/tests/test_auth.py

```diff
@@ -248,7 +248,7 @@ def test_passwd(self):
         }
         response = self.client.post(url, submit,
                                     HTTP_AUTHORIZATION='token {}'.format(token))
-        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.status_code, 401)
```

Comment:
I'd try to keep the responses the same as they were before for compatibility's sake. Is there a specific reason why we want to change this from a 400 to a 401?